### ISSUE TITLE
Images rendered have a bias on the left side

### ISSUE THREAD

**Issue by vvavepacket** [brh55/react-native-masonry#71]
## Expected Behavior

I expect Masonry to balance the images loaded on the screen. Balance means equal height on all columns. 

## Current Behavior

For every Masonry component rendered, I observed a bias toward the first column. It tends to load most of the images on the left side/1st column.

## Possible Solution

Maybe do some calculation for final rendering to balance the images so that it does not gravitate toward any specific column

## Steps to Reproduce

![image](https://user-images.githubusercontent.com/8295642/34463602-1e870d32-ee2f-11e7-819b-62e763388474.png)
![image](https://user-images.githubusercontent.com/8295642/34463603-227ee3ba-ee2f-11e7-9ee0-7a4a6d86afd1.png)
![image](https://user-images.githubusercontent.com/8295642/34463608-32777a20-ee2f-11e7-8664-0b494708c514.png)

**Comment by brh55** [comment]
It's currently by design, if you have the sorted property, it will load per order. If it's not on, then it's just loading columns as first come first serve. Doesn't take account for any heights

**Comment by vvavepacket** [comment]
Is there any hack or trick to remove bias?

**Comment by brh55** [comment]
Not from a user's perspective. Within the codebase, you could put together something that updates the an aggregate of each column height per an image, and inserts it accordingly.

**Comment by brh55** [comment]
@vvavepacket 

Currently have a working implementation within the `priority-option` branch. But looking to improve on its implementation and test it more thoroughly. Feel free to test it out by setting the priority attribute to "balance",`priority={'balance'}`.

**Comment by alpamys-qanybet** [comment]
here what I have done, just downloaded the source code and use it instead of package.
and in Masonry.js

```
// containMatchingUris :: ([brick], [brick]) -> Bool
const containMatchingUris = (r1, r2) => false; //isEqual(r1.map(brick => brick.uri), r2.map(brick => brick.uri));
```
I found out that during initialization it renders in a normal way, but caching makes it not work on Android, so I just tested to disable cache and it works.

**Comment by brh55** [comment]
Will be included in v0.5.0.
